app/src/main/res/xml/rowkeys_kabyle1.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+/*
+**
+** Copyright 2012, The Android Open Source Project
+**
+** Licensed under the Apache License, Version 2.0 (the "License");
+** you may not use this file except in compliance with the License.
+** You may obtain a copy of the License at
+**
+**     http://www.apache.org/licenses/LICENSE-2.0
+**
+** Unless required by applicable law or agreed to in writing, software
+** distributed under the License is distributed on an "AS IS" BASIS,
+** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+** See the License for the specific language governing permissions and
+** limitations under the License.
+*/
+-->
+
+<merge
+    xmlns:latin="http://schemas.android.com/apk/res-auto"
+>
+    <switch>
+        <case latin:showNumberRow="true">
+            <Key
+                latin:keySpec="a"
+                latin:moreKeys="!text/morekeys_a" />
+		<!-- -->
+            <Key
+                latin:keySpec="z"
+                latin:moreKeys="!text/morekeys_z" />
+		<!-- -->
+            <Key
+                latin:keySpec="e"
+                latin:moreKeys="!text/morekeys_e" />
+		<!-- -->
+            <Key
+                latin:keySpec="r"
+                latin:moreKeys="!text/morekeys_r" />
+		<!-- -->
+            <Key
+                latin:keySpec="t"
+                latin:moreKeys="!text/morekeys_t" />
+		<!-- -->
+            <Key
+                latin:keySpec="y"
+                latin:moreKeys="!text/morekeys_y" />
+		<!-- -->
+            <Key
+                latin:keySpec="u"
+                latin:moreKeys="!text/morekeys_u" />
+		<!-- -->
+            <Key
+                latin:keySpec="i"
+                latin:moreKeys="!text/morekeys_i" />
+		<!-- -->
+            <Key
+                latin:keySpec="&#x025B;"
+                latin:moreKeys="!text/morekeys_o" />
+		<!-- -->
+            <Key
+                latin:keySpec="&#x0263;" />
+		<!-- -->
+        </case>
+        <default>
+            <Key
+                latin:keySpec="a"
+                latin:keyHintLabel="1"
+                latin:additionalMoreKeys="1"
+                latin:moreKeys="!text/morekeys_a" />
+            <Key
+                latin:keySpec="z"
```

Comment:
Do not redefine an AZERTY layout. Look how French language is implemented.

Reply:
Ok, nice !